awscli-bundle/install

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
```

Comment:
Did you mean to check-in the aws-cli and it's dependencies?

Reply:
yeah we should probably not do this